src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -287,6 +287,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		)
 		#endif
 	}()
+	
+	lazy var healthCertificateService: HealthCertificateService = HealthCertificateService(
```

Comment:
```suggestion
	private lazy var healthCertificateService: HealthCertificateService = HealthCertificateService(
```

Reply:
re-added `private` after now injecting the service to the notification manager